phoebe/atmospheres/passbands.py

```diff
@@ -295,8 +295,8 @@ def __init__(self, ptf=None, pbset='Johnson', pbname='V', effwl=5500.0,
         # self.nntree = dict()              # nearest neighbor tree for blending and extrapolation
         # self.indices = dict()             # nearest neighbor indices for blending and extrapolation
         # self.ics = dict()                 # inferior corners for blending and extrapolation
-        self.blending_region = dict()     # blending regions
-        self.mapper = dict()              # mapping from model atmosphere axes to interpolation axes
+        # self.blending_region = dict()     # blending regions
```

Comment:
I'm guessing this is a temporary change to disable blending for now?

Reply:
nope, that's a permanent change; blending regions are now replaced by unit hypercubes in cndpolator. My upcoming commit will remove these altogether.